packages/ckeditor5-html-support/docs/features/general-html-support.md

```diff
@@ -6,7 +6,7 @@ updated_at: 2021-06-14
 
 # General HTML Support
 
-The General HTML Support ("GHS") feature allows developers to easily enable HTML features that are not explicitly supported by any other dedicated CKEditor 5 plugins. It acts similarly to the {@link @ckeditor4 /guide/dev/acf/README Advanced Content Filtering} feature from CKEditor 4, letting the user expand the list of supported HTML markup in the content.
+The General HTML Support ("GHS") feature allows developers to easily enable HTML features that are not explicitly supported by any other dedicated CKEditor 5 plugins. It acts similarly to the {@link @ckeditor4 guide/dev/deep_dive/advanced_content_filter/README Advanced Content Filtering} feature (ACF) from CKEditor 4, a feature that filters incoming HTML content by transforming and deleting disallowed elements, attributes, classes and styles. The GHS feature allows for adding this kind of markup to the source and prevents it from being filtered from the editor window and the output.
```

Comment:
```suggestion
The General HTML Support ("GHS") feature allows developers to easily enable HTML features that are not explicitly supported by any other dedicated CKEditor 5 plugins. It acts similarly to {@link @ckeditor4 /guide/dev/acf/README Advanced Content Filter} (ACF) from CKEditor 4, a feature that filters incoming HTML content by transforming and deleting disallowed elements, attributes, classes and styles. GHS allows for adding this kind of markup to the source and prevents it from being filtered from the editor window and the output.
```

Reply:
The first link is kinda intro, so let's use it - deep dive is more advanced. The name of the feature in CKE4 is Advanced Content Filter (ACF).